train.py

```diff
@@ -0,0 +1,27 @@
+# %%
+import os
+import json
+
+import cssdata
+
+# %% import data and dataframe
+
+cwd = os.getcwd()  # get current working directory
+data_dir = os.path.join(cwd, "rawdata")  # define the path for the rawdata
+inputfile = "input.json"  # name of the `input.json`
+inputfile_dir = os.path.join(cwd, inputfile)  # define the path for the `input.json`

```

Comment:
This part should be a command-line argument. Use https://docs.python.org/3/library/argparse.html
Get the working directory and the name of the input JSON file. So you don't need all these superfluous code from lines 9 - 12.